hlsdl.go

```diff
@@ -43,14 +44,19 @@ type DownloadResult struct {
 	SeqId uint64
 }
 
-func New(hlsURL string, headers map[string]string, dir string, workers int, enableBar bool) *HlsDl {
+func New(hlsURL string, headers map[string]string, dir string, workers int, enableBar bool, filename string) *HlsDl {
+	if filename == "" {
+		filename = getTimestamp()
```

Comment:
@ernesto27  Instead of returning a timestamp. Why should not return a filename?

Reply:
Then we don't need to append `.ts` after the filename.

Reply:
Yes, that is right.
i just uploaded a new commit with these changes